### DISCUSSION TITLE
Is <FooterRegion> available?

### DISCUSSION THREAD

**Discussion by umeshvenkat** [blazority/support#50] [Ideas]
Application layout supports required components like 'MainContainer', 'HeaderRegion'. Not finding a footer component. Is it available? Is there any specific approach to be followed to add footer the layout.

**Comment by karan-kang** [comment]
Hi @umeshvenkat.

We will add this support for this in upcoming v0.10.0 release.
https://github.com/blazority/support/issues/57
